test/conformance/testing/include/uur/fixtures.h

```diff
@@ -501,41 +500,22 @@ struct urMultiDeviceMemBufferQueueTest : urMultiDeviceMemBufferTest {
         for (const auto &queue : queues) {
             EXPECT_SUCCESS(urQueueRelease(queue));
         }
-        UUR_RETURN_ON_FATAL_FAILURE(urMultiDeviceMemBufferTest::TearDown());
+        UUR_RETURN_ON_FATAL_FAILURE(urMemBufferQueueTest::TearDown());
     }
 
     std::vector<ur_queue_handle_t> queues;
 };
 
-struct urMemBufferQueueTest : urQueueTest {
```

Comment:
I spotted this could be a  redundant code so wanted to address it, there is other redundant code, also added by this PR, that need to refactor the tests a little to eliminate it.